app/src/pages/inside/common/durationBlock/durationBlock.jsx

```diff
@@ -82,13 +83,26 @@ export class DurationBlock extends Component {
       approxTime: PropTypes.number,
     }).isRequired,
     status: PropTypes.string,
+    id: PropTypes.number,
+    activeProject: PropTypes.string,
     itemNumber: PropTypes.number,
   };
   static defaultProps = {
     type: '',
     status: '',
+    id: null,
+    activeProject: '',
     itemNumber: null,
   };
+
+  componentDidMount() {
+    this.intervalId = setInterval(() => this.fetchLaunch(this.props.id), 5000);
```

Comment:
I think it's better to do this on the page level. Moreover, I see in the api definition that this endpoint can work with an array of ids. Looks like you don't have to send a separate request for every launch.